source/usr/lib/mcvirt/virtual_machine/hard_drive/drbd.py

```diff
@@ -270,25 +270,6 @@ def getSize(self):
         self._ensure_exists()
         return self._get_logical_volume_size(self._getLogicalVolumeName(self.Drbd_RAW_SUFFIX))
 
-    def get_remote_object(self):
-        """Return a instance of the virtual machine object
-        on the machine that the VM is registered
-        """
-        if self.vm_object.isRegisteredLocally():
-            return self
-        elif self.vm_object.isRegisteredRemotely():
-            cluster = self._get_registered_object('cluster')
-            remote_node = cluster.get_remote_node(self.vm_object.getNode())
-            remote_vm_factory = remote_node.get_connection('virtual_machine_factory')
-            remote_vm = remote_vm_factory.getVirtualMachineByName(self.vm_object.get_name())
-            remote_node.annotate_object(remote_vm)
-            remote_hard_drive_factory = remote_node.get_connection('hard_drive_factory')
-            remote_hard_drive = remote_hard_drive_factory.getObject(remote_vm, self.disk_id)
-            remote_node.annotate_object(remote_hard_drive)
-            return remote_hard_drive
-        else:
-            raise VmNotRegistered('The VM is not registered on a node')
-
```

Comment:
This was added, but I realised that the method had already been defined in the base class

On 24 Aug 2016, at 10:56, joe-singleton-itdev <notifications@github.com<mailto:notifications@github.com>> wrote:


In source/usr/lib/mcvirt/virtual_machine/hard_drive/drbd.py<https://github.com/ITDevLtd/MCVirt/pull/222#discussion_r76027816>:

> -        """
> -        if self.vm_object.isRegisteredLocally():
> -            return self
> -        elif self.vm_object.isRegisteredRemotely():
> -            cluster = self._get_registered_object('cluster')
> -            remote_node = cluster.get_remote_node(self.vm_object.getNode())
> -            remote_vm_factory = remote_node.get_connection('virtual_machine_factory')
> -            remote_vm = remote_vm_factory.getVirtualMachineByName(self.vm_object.get_name())
> -            remote_node.annotate_object(remote_vm)
> -            remote_hard_drive_factory = remote_node.get_connection('hard_drive_factory')
> -            remote_hard_drive = remote_hard_drive_factory.getObject(remote_vm, self.disk_id)
> -            remote_node.annotate_object(remote_hard_drive)
> -            return remote_hard_drive
> -        else:
> -            raise VmNotRegistered('The VM is not registered on a node')
> -


How come this function was removed?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ITDevLtd/MCVirt/pull/222/files/7bd788df35c290cafa25fee74c74c473f89c8bca#r76027816>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABNSVs_ISXhSSH7UhxjqI8hDv8hcJmYlks5qjBUvgaJpZM4Jrz9u>.